src/frontend/README.md

```diff
@@ -1,7 +1,128 @@
-# frontend
+# frontend service
 
-Run the following command to restore dependencies to `vendor/` directory:
+The **frontend** service is responsible for rendering the UI for the store's website.
+It serves as the main entry point for the application routing requests to their
+appropriate backend services.
+The application uses Server Side Rendering (SSR) to generate HTML consumed by
+the clients, which could be web browsers, web crawlers, mobile clients or something else.
 
-```sh
-dep ensure --vendor-only
+## OpenTelemetry features
+
+| Auto-Instrumented tracing | Custom spans | Custom span attributes | Metrics | Logs |
+|---------------------------|--------------|------------------------|---------|------|
+| X                         |              | X                      |         |      |
+
+## OpenTelemetry instrumentation
+
+### Initialization
+
+The OpenTelemetry SDK is initialized in `main` using the `InitTraceProvider` function.
+
+```go
```

Comment:
Is there a way we could include code samples in a more robust way?

Reply:
I'm happy to help figure out a better way. Do you have any suggestions on what we should do?